talker/src/talker.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright (c) 2019, ROSIN-project
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+1. Redistributions of source code must retain the above copyright notice, this
+list of conditions and the following disclaimer.
+
+2. Redistributions in binary form must reproduce the above copyright notice,
+this list of conditions and the following disclaimer in the documentation
+and/or other materials provided with the distribution.
+
+3. Neither the name of the copyright holder nor the names of its
+contributors may be used to endorse or promote products derived from
+this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+/**
+ * This tutorial demonstrates simple sending of messages over the ROS system 
+ * using the rxros interface
+ */
+
+#include <rxros/rxros.h>
+#include "std_msgs/String.h"
+
+using namespace rxcpp::operators;
+using namespace rxros::operators;
+
+std_msgs::String mk_msg (const std::string& s)  // the missing factory
```

Comment:
> // the missing factory

This comment is unclear: why is this missing? Is this something that would be clear to people used to RxCPP? Is something still missing? Or *was* it missing?

Suggestion: either explain why it is/was missing, or remove.

Reply:
I removed the comment.  I agree that out of context it is just confusing, especially in a tutorial. 

Context: I am with OP on this one: https://github.com/ros/ros_comm/issues/148
When you are a functional programmer, these fast constructors, that do not require several lines of assignments are really your basic tool.  Perhaps because one constructs new objects much more often than in the imperative style.